src/javascript/_common/base/login.js

```diff
@@ -14,14 +16,17 @@ const Login = (() => {
     const loginUrl = () => {
         const server_url = localStorage.getItem('config.server_url');
         const language   = getLanguage();
+        const signup_device      = LocalStore.get('signup_device') || /Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(navigator.userAgent) ? 'mobile' : 'desktop';
+        const date_first_contact = moment(LocalStore.get('date_first_contact'), 'YYYY-mm-dd').valueOf();
```

Comment:
if the localstore value is undefined I believe it will return today's date, no?

Reply:
No, check this bin out. 

https://jsbin.com/bokitemavi/edit?js,output